README.md

```diff
@@ -22,7 +21,26 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Basic Configuration
+In order to capture and report all deprecation warnings at the end of the test run, add the following to your test setup
+```ruby
+require 'active_support/deprecation_test_helper'
+ActiveSupport::DeprecationTestHelper.configure
+```
+
+### Whitelisting Deprecation Warnings
```

Comment:
We are starting to intentionally make an effort to move away from using terms like Whitelist and Blacklist, and instead choose different terminology that still captures the idea (e.g. allow lists, permitted lists, and exclusion lists).  In this case though, it may be more clear as `Ignore Deprecation Warnings` or `Skip Alerting on Specific Deprecation Warnings` ?

Reply:
Thank you for the reminder on this, I've updated it to `Expected Deprecation Warnings`